File: src/main/java/com/aquasecurity/plugins/trivy/model/MisconfSummary.java
package com.aquasecurity.plugins.trivy.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class MisconfSummary {
    @JsonProperty("Successes")
    public int successes;
    @JsonProperty("Failures")
    public int failures;
    @JsonProperty("Exceptions")
    public int exceptions;
}

File: src/main/kotlin/com/aquasecurity/plugins/trivy/model/MisconfSummary.kt
package com.aquasecurity.plugins.trivy.model

import com.fasterxml.jackson.annotation.JsonProperty

class MisconfSummary {
    @JsonProperty("Successes")
    var successes: Int = 0

    @JsonProperty("Failures")
    var failures: Int = 0

    @JsonProperty("Exceptions")
    var exceptions: Int = 0
}